lib/glific/flows/broadcast.ex

```diff
@@ -186,7 +187,11 @@ defmodule Glific.Flows.Broadcast do
         contacts,
         fn contact ->
           Repo.put_process_state(contact.organization_id)
-          response = FlowContext.init_context(flow, contact, @status, opts)
+
```

Comment:
we are just starting a broadcast here, why are we checking for paused status? Are we pausing at the contact level?